tools/internal_ci/linux/grpc_bazel_rbe.sh

```diff
@@ -30,6 +30,7 @@ python3 tools/run_tests/python_utils/bazel_report_helper.py --report_path bazel_
 
 bazel_rbe/bazel_wrapper \
   --bazelrc=tools/remote_build/linux_kokoro.bazelrc \
+  --output_user_root=/tmpfs/bazel \
```

Comment:
nice find and good short term fix, but we should probably address the issue in a more systemic way (we have many more bazel builds than just this).